app/views/custom/layouts/_footer.html.erb

```diff
@@ -26,9 +26,7 @@
     <div class="small-12 medium-8 column">
       <%= t("layouts.footer.copyright", year: Time.current.year) %>&nbsp;|
       <ul class="no-bullet inline-block">
-        <li class="inline-block"><%= link_to t("layouts.footer.privacy"), page_path("privacy") %>&nbsp;|</li>
-        <li class="inline-block"><%= link_to t("layouts.footer.conditions"), page_path("conditions") %>&nbsp;|</li>
-        <li class="inline-block"><%= link_to t("layouts.footer.accessibility"), page_path("accessibility") %></li>
```

Comment:
Maybe we could add a condition so the link is only shown if the page exists? :thinking: That way, if they decide they need to add these pages again (maybe a legal department might mention it), we won't have to change the code. And we could even add a test for this case :smile:.